src/style.css

```diff
@@ -0,0 +1,89 @@
+body {
+  margin: 0;
+}
+
+.snake {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+}
+
+.block {
+  display: flex;
+  flex-wrap: wrap;
```

Comment:
```suggestion
```
No need for this here. You only have one item inside - number. It cannot be wrapped in any way